Tutorials/Editors-Manual/Getting-Started-With-Umbraco/Creating-Saving-and-Publishing-Content/index.md

```diff
@@ -98,6 +98,48 @@ To set up scheduled unpublishing, follow these steps:
     ![Schedualed unpublishing.](images/Schedule_Unpublishing_v9.png)
 5. Select **Schedule**.
 
+## Comparing Content between environments
+
+:::note
+Content comparison is currently available in all Umbraco Cloud projects running Umbraco Deploy 9.3+ for Umbraco 9 and Umbraco Deploy 4.5+ for Umbraco 8.
+:::
+
+Compare Content allows to preview content changes before transferring them to another environment. This is helpful to ensure that the correct updates are transferred when working with content in multiple environments.
+
+You can see the **Summary Information** and **Field Comparision** values to understand what will change if you proceed to transfer the content upstream or try restoring content to the current environment.
+
+To compare content between environments, follow these steps:
+
+1. Navigate to the page you want to compare.
+2. Select the arrow next to the **Save and Publish** button. Alternatively, you can right-click the content node or click the **Actions** drop-down.
+3. Select **Compare**.
+
+    ![Compare option](images/Compare_option.png)
+4. In the **Compare** window, you can do the following:
+    a. **Choose the workspace** from the drop-down field.
+    b. View the **Summary information**.
+    c. In the **Field Comparision** table, view the differences between the versions in the two workspaces at the node level of each field.
+    d. Proceed to transfer the content using the **Queue for transfer** or **Transfer now** options.
+    e. Restore the content from the upstream environment using the **Partial restore** option.
+    f. Click **Close** to continue editing the content node.
```

Comment:
Suggestion:
Instead of making a list within the list, wouldn't it make sense to pull the items out like this?

```suggestion
4.  **Choose the workspace** from the drop-down field.
5.  View the **Summary information**.
5. . In the **Field Comparision** table, view the differences between the versions in the two workspaces at the node level of each field.
6. Proceed to transfer the content using the **Queue for transfer** or **Transfer now** options.
7. Restore the content from the upstream environment using the **Partial restore** option.
8. Click **Close** to continue editing the content node.
```